test/gst-msdk/encode/encoder.py

```diff
@@ -36,7 +36,7 @@ def gen_output_opts(self):
       if self.codec in ["mpeg2"]:
         opts += " qpi={mqp} qpp={mqp} qpb={mqp}"
       else:
-        opts += " qpp={qp}"
+        opts += " qpi={qp} qpp={qp} qpb={qp}"
```

Comment:
It looks like the same {qb} for qpi/qpp/qpb.  But from sample_encode output
 [-qpi]                   - constant quantizer for I frames (if bitrace control method is CQP). In range [1,51]. 0 by default, i.e.no limitations on QP.
   [-qpp]                   - constant quantizer for P frames (if bitrace control method is CQP). In range [1,51]. 0 by default, i.e.no limitations on QP.
   [-qpb]                   - constant quantizer for B frames (if bitrace control method is CQP). In range [1,51]. 0 by default, i.e.no limitations on QP.

It looks like qpi/qpp/qpb can be different values. if gst-msdk need they be same ?

Reply:
no, gst-msdk doesn't need the same value for QPx

Reply:
The MSDK library recently changed how QPx=0 is handled, so the sample_encode help is misleading (and ambiguous to begin with).  MSDK used to set QPx=1 when user sent 0, now it sets to different defaults and changes our test results if we aren't explicit.  We are working to get MSDK to make the behavior more clear.  Either way, let's be explicit in the test so we're not at their mercy.  Agree?

It's ok if user explicitly sets QPx to same values, but it is not required.  On ffmpeg side, they use an ipratio/pbratio to set different QPI/QPB relative to QPP.  But in gst-vaapi, they set all QPx the same by default (specified by init-qp prop).  This patch just sets them all explicitly to the same value so gst-msdk can closer match gst-vaapi behavior and to avoid the unspecified behavior by MSDK library.